samples/sensor/accel_stream/src/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/sensor.h>
+
+#define ACCEL_TRIGGERS						\
+	{ SENSOR_TRIG_DRDY, SENSOR_STREAM_DATA_INCLUDE },	\
+	{ SENSOR_TRIG_TAP, SENSOR_STREAM_DATA_NOP }
+#DEFINE ACCEL_ALIAS(id) DT_ALIAS(CONCAT(accel, id))
+#define ACCEL_IODEV_SYM(id) CONCAT(accel_iodev, id)
+#define ACCEL_IODEV_PTR(id) &ACCEL_IODEV_SYM(id)
+#define ACCEL_DEFINE_IODEV(id)					\
+	SENSOR_DT_STREAM_IODEV(				\
```

Comment:
```suggestion
	SENSOR_DT_STREAM_IODEV(					\
```

Reply:
This looks ok to me, wonder why it looks funny in github